client/client-kv.c

```diff
@@ -0,0 +1,215 @@
+#include <stdlib.h>
```

Comment:
Oh, could not remember clearly, we used this client for testing server-kv lib, and it showed up as size_t could not be found, that is why <stdlib.h> is included here~~~pls correct if I made it wrong, as actually, stdio.h should have already included defining of "size_t" ~~~